orion-ui/src/router/routes.ts

```diff
@@ -11,6 +11,7 @@ export const routes = {
   flowRun: (id: string) => ({ name: 'flow-run', params: { id } }) as const,
   flowRunCreate: (deploymentId: string) => ({ name: 'flow-run.create', params: { deploymentId } }) as const,
   flowRuns: () => ({ name: 'flow-runs' }) as const,
+  flowRunsOld: () => ({ name: 'flow-runs-old' }) as const,
```

Comment:
Same question about whether we need to keep this but thanks for considering it!

Reply:
Happy to remove it if its requested though!